packages/indexer-cli/src/command-helpers.ts

```diff
@@ -167,10 +167,13 @@ export async function validatePOI(poi: string | undefined): Promise<string | und
   return poi
 }
 
-export function outputColors(
-  print: GluegunPrint,
-  outputFormat: 'table' | 'json' | 'yaml',
-): void {
+export function outputFormatting(print: GluegunPrint, outputFormat: string): void {
+  if (!['json', 'yaml', 'table'].includes(outputFormat)) {
+    print.error(`Invalid output format "${outputFormat}"`)
+    process.exitCode = 1
+    return
+  }
+
```

Comment:
What's the motivation for this change?
Wouldn't it be safer to rely on the type checker for validating the possible values for the `outputFormat` parameter instead of checking it during runtime?

Reply:
Sorry, I should clarify that by "safer," I mean type-safe.

The TS compiler would prevent invalid input from being passed to the function in the previous code. I believe this promotes a better developer experience because it removes the need for keeping the validation code inside the function, as it would prevent us from ever sending invalid strings to it. I could summarize this safety claim as "having a compile-time error is preferable to having a runtime error."

Nonetheless, we still need to sanitize the CLI inputs. On this matter, I would suggest that CLI options be validated right after the program starts rather than when the code requires them. This way, we wouldn't worry about invalid values being carried downstream while the program would execute in vain.

Reply:
Ideally, I'd suggest creating a parser function with a `string -> OutputFormat` signature, where `OutputFormat` is an enum of possible table formats. We would then call this function on every command that accepts the table format input as soon as that value is available to us.

Reply:
I've updated to a `OutputFormat` enum and parser function for the command inputs, it doesn't utilize the TS compiler with type-safe inputs because I wanted to make it clearly return something undefined for invalid inputs, but let me know if the function should include the possible values as inputs?